gcp/table_gcp_vertex_ai_model.go

```diff
@@ -0,0 +1,321 @@
+				Description: "User provided version aliases so that a model version can be referenced via alias.",
+			},
+			{
+				Name:        "predict_schemata",
+				Type:        proto.ColumnType_JSON,
+				Description: "The schemata that describe formats of the model's predictions and explanations.",
+			},
+			{
+				Name:        "metadata",
+				Type:        proto.ColumnType_JSON,
+				Description: "An additional information about the model; the schema of the metadata can be found in metadata_schema_uri, immutable.",
+			},
+			{
+				Name:        "supported_export_formats",
+				Type:        proto.ColumnType_JSON,
+				Description: "The formats in which this model may be exported. If empty, this model is not available for export.",
+			},
+			{
+				Name:        "container_spec",
+				Type:        proto.ColumnType_JSON,
+				Description: "The specification of the container that is to be used when deploying this model.",
+			},
+			{
+				Name:        "supported_deployment_resources_types",
+				Type:        proto.ColumnType_JSON,
+				Description: "The configuration types this model supports for deployment.",
+			},
+			{
+				Name:        "supported_input_storage_formats",
+				Type:        proto.ColumnType_JSON,
+				Description: "The formats this Model supports in BatchPredictionJob.input_config.",
+			},
+			{
+				Name:        "supported_output_storage_formats",
+				Type:        proto.ColumnType_JSON,
+				Description: "The formats this Model supports in BatchPredictionJob.output_config.",
+			},
+			{
+				Name:        "deployed_models",
+				Type:        proto.ColumnType_JSON,
+				Description: "The pointers to DeployedModels created from this Model.",
+			},
+			{
+				Name:        "explanation_spec",
+				Type:        proto.ColumnType_JSON,
+				Description: "The default explanation specification for this Model.",
+			},
+			{
+				Name:        "labels",
+				Type:        proto.ColumnType_JSON,
+				Description: "The labels with user-defined metadata to organize your Models.",
+			},
+			{
+				Name:        "encryption_spec",
+				Type:        proto.ColumnType_JSON,
+				Description: "Customer-managed encryption key spec for a Model.",
+			},
+			{
+				Name:        "model_source_info",
+				Type:        proto.ColumnType_JSON,
+				Description: "Source of a model.",
+			},
+			{
+				Name:        "original_model_info",
+				Type:        proto.ColumnType_JSON,
+				Description: "If this model is a copy of another model, this contains info about the original.",
+			},
+			// Standard columns for integration with Steampipe
+			{
+				Name:        "title",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("DisplayName"),
+				Description: "Title of the model, usually the same as display_name.",
+			},
+			{
+				Name:        "tags",
+				Type:        proto.ColumnType_JSON,
+				Transform:   transform.FromField("Labels"),
+				Description: "A map of tags for the resource, represented as a JSON object.",
```

Comment:
@Priyanka-Chatterjee-2000  I believe we should add `standard gcp columns` in this table.